mesa/time.py

```diff
@@ -31,6 +31,15 @@
 from mesa.agent import Agent, AgentSet
 from mesa.model import Model
 
+# Throw module-level deprecation warning
+warnings.warn(
+    "The time module and all its Schedulers are deprecated and will be removed in a future version. "
```

Comment:
Does this mean 3.1 or 4.0?

Reply:
I would not put a timeline on its removal. 

Just deprecation is enough. We don't need to worry about maintaining and testing anymore, but it might be convenient to keep it around for a while to make it easier for people to migrate models.

Reply:
Yeah good question, I'm not sure about that. I'm nudging towards 3.1 or 3.2.

Another option is leaving it as a warning for now, and flipping it to an Error at some point.

Reply:
I wrote the comment before I read Jan's comment, but I agree with him. We don't support it anymore and it will get removed.